External docs links

jameshadfield opened the pull request:
This is a small PR to improve duplicated (or overly similar) documentation pages across different sites using the ability for nextstrain.org/docs to have external links in the sidebar (see #191, on which this PR is build). I also add a few more external links to pages which seemed to be immediately useful for users of nextstrain.org/docs. 
I realise that a major shake-up of how documentation will be orgainsed is under discussion (this is the not-yet-publicised but google indexed docs.nextstrain.org), and that questions such as whether tutorials should live in the main nextstrain.org docs (currently nextstrain.org/docs) or within augur are currently unresolved.

# Changes to sections within nextstrain.org/docs

## Getting Started

No changes

## Bioinformatics

#### Augur commands

nextstrain.org/docs/bioinformatics/augur-commands was a stub with a link to https://docs.nextstrain.org/projects/augur/en/stable/usage/cli/cli.html.
I've therefore replaced it with an external link straight from the sidebar.

#### FAQ

Added an external link to the RTD hosted FAQ.

## Tutorials.

A general observation is that our tutorials are actually a combination of a tutorial and a bunch of how-to documentation combined.
Separating those out is a project for another day!

### Zika: (nextstrain.org/docs/tutorials/zika vs https://docs.nextstrain.org/projects/augur/en/stable/tutorials/zika_tutorial.html)
These tutorials are very diverged (perhaps they don't share a common ancestor?!?).
I find the nextstrain.org version easier to follow, as the RTD version spends a lot of time introducing the concept of augur commands, and have therefore left it untouched.

### TB (nextstrain.org/docs/tutorials/tb vs https://docs.nextstrain.org/projects/augur/en/stable/tutorials/tb_tutorial.html)

These are very similar; the main differences being that the RTD version assumes you have installed augur, but also seems to be the most up-to-date (for instance using `--output-node-data` instead of `--output`).
I've removed nextstrain.org/docs/tutorials/tb and added a sidebar entry (and redirect) to the RTD version.


#### Defining clades

Untouched. I am in favor of the changes in https://github.com/nextstrain/nextstrain.org/pull/170/files but think they should be moved to the augur docs.
Will cross-comment on that PR.

## Contributing

Unchanged

## Narratives

This content is more up-to-date (and better) in the auspice docs.
I've therefore removed this from nextstrain.org/docs and added two external links, one within the Visualisation section (an introduction to narratives) and one in the tutorials section (a step-by-step on how to write).
A redirect is added from the now removed page to the narratives introduction.


## Visualisation

### Download data & interpreting the map

Left untouched. These should be moved to the auspice docs site.

## Extra links

Added links to the nextstrain.org/docs sidebar pointing to the following pages on the auspice docs site:

* Interactive Narratives (see above)
* Drag & Drop metadata (CSV / TSV)
* How to view multiple trees


# Brief notes for work outside of this PR

* Update the visualisation section of the zika & TB tutorials to use http://auspice.us
* Remove (?) the RTD Zika tutorial
* Add a section to the RTD MTb tutorial explaining what you need to install to run the tutorial.
* Propose moving the "defining clades" content to augur's RTD and linking to that. See #170.
* Move the download data & interpreting the map pages to auspice's docs.

eharkins commented:
Just a note to rebase on master before merging to get groups doc added in #193

jameshadfield commented:
Thanks @huddlej and @eharkins 💯 

> It would be nice if the `README.md` for the static-site mentioned that these links are defined in that file, so we will know where to look in the future.

Absolutely 👍  -- updated this PR with this.

> sometimes flashes the nextstrain.org 404 page temporarily before going to the redirect path

We've had issues with 404 flashes in the past in various places in Gatsby (and so have others). Since we may shortly be moving off Gatsby I didn't explore this. 

> Just a note to rebase on master before merging to get groups doc added in #193

Thanks! Done.

> I vote for tutorials living in augur + auspice docs (so that they can be updated in sync with changes to each of those projects) so I would be in favor of consolidating the zika tutorial into https://nextstrain-augur.readthedocs.io/en/stable/tutorials/zika_tutorial.html

I agree. I've created https://github.com/nextstrain/augur/issues/594 to update the zika tutorial (and then we can add an external link here).

> I'm excited to discuss the other docs points in more detail elsewhere.

Me too!

> /docs/getting-started/container-installation -> https://docs.nextstrain.org/projects/cli/en/latest/installation/#docker

Agreed -- added to the PR.

> /docs/contributing/development -> https://github.com/nextstrain/.github/blob/master/CONTRIBUTING.md

Good find! Added to the PR.

> /docs/bioinformatics/customizing-a-build -> maybe a whole section on this with external links to things like https://nextstrain-augur.readthedocs.io/en/stable/faq/lat_longs.html unless the intention is to write a comprehensive how-to on customizing a build (in which case it would make sense to go in the augur docs and link from here anyway)

Agreed -- this page shouldn't be up as it contains no information. I've removed it, updated the links to it from other docs pages, and created a redirect to the augur docs just in case.
